src/Screens/Onboarding/DisplayName.tsx

```diff
@@ -43,46 +43,61 @@ const DisplayName = ({ navigation }: Props) => {
   return (
     <OnboardingBackground testID="onboarding.displayName.view">
       <Card style={styles.card}>
-        <Message style={styles.title} id="onboarding.displayName.title" />
-        <NamedInputField
-          autoCapitalize="none"
-          style={styles.nickNameInput}
-          name="onboarding.displayName.inputTitle"
-          onChangeText={setDisplayName}
-          autoCompleteType="off"
-          value={displayName}
-          testID="onboarding.displayName.inputTitle"
-        />
-        <Message style={styles.bodyText} id="onboarding.displayName.bodyText" />
-        <Button
-          style={isValidDisplayName ? styles.nextButton : styles.notValidButton}
-          onPress={navigateToEmailScreen}
-          messageId="onboarding.displayName.nextButton"
-          badge={require('../images/arrow.svg')}
-          testID="onboarding.displayName.nextButton"
-          disabled={!isValidDisplayName}
-        />
-        <Button
-          messageId="onboarding.signUp.back"
-          onPress={goBack}
-          noShadow={true}
-          style={{ backgroundColor: colors.lightestGray }}
-        />
+        <RN.View>
+          <Message style={styles.title} id="onboarding.displayName.title" />
+          <NamedInputField
+            autoCapitalize="none"
+            style={styles.nickNameInput}
+            name="onboarding.displayName.inputTitle"
+            onChangeText={setDisplayName}
+            autoCompleteType="off"
+            value={displayName}
+            testID="onboarding.displayName.inputTitle"
+          />
+          <Message
+            style={styles.bodyText}
+            id="onboarding.displayName.bodyText"
+          />
+        </RN.View>
+        <RN.View style={styles.buttonContainer}>
+          <Button
+            messageId="onboarding.signUp.back"
+            onPress={goBack}
+            noShadow={true}
+            style={styles.backButton}
+          />
+
+          <Button
+            style={
+              isValidDisplayName ? styles.nextButton : styles.notValidButton
+            }
+            badgeStyle={styles.badgeStyle}
+            onPress={navigateToEmailScreen}
+            messageId="onboarding.displayName.nextButton"
+            badge={require('../images/arrow-right.svg')}
+            testID="onboarding.displayName.nextButton"
+            disabled={!isValidDisplayName}
+          />
+        </RN.View>
       </Card>
     </OnboardingBackground>
   );
 };
 
 const styles = RN.StyleSheet.create({
   card: {
+    flex: 1,
+    flexDirection: 'column',
+    justifyContent: 'space-between',
+    marginVertical: 20,
```

Comment:
are we using 20 here? multiples of 8 are best for high DPI stuff. But if design requires 20 then 20 is the best probably.